src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        Figure[] figuresArray = new Figure[6];

```

Comment:
```suggestion
        Figure[] figuresArray = new Figure[6];
```
6 - better make constant